src/debug/webgl-debug.js

```diff
@@ -92,31 +92,31 @@ var glValidEnumContexts = {
   'getTexParameter': {2: { 0:true, 1:true }},
   'texParameterf': {3: { 0:true, 1:true }},
   'texParameteri': {3: { 0:true, 1:true, 2:true }},
-  'texImage2D': {
-     9: { 0:true, 2:true, 6:true, 7:true },
-     6: { 0:true, 2:true, 3:true, 4:true }
-  },
-  'texSubImage2D': {
-    9: { 0:true, 6:true, 7:true },
-    7: { 0:true, 4:true, 5:true }
-  },
+  //'texImage2D': {
+  //   9: { 0:true, 2:true, 6:true, 7:true },
+  //   6: { 0:true, 2:true, 3:true, 4:true }
+  //},
+  //'texSubImage2D': {
+  //  9: { 0:true, 6:true, 7:true },
+  //  7: { 0:true, 4:true, 5:true }
+  //},
```

Comment:
Let's not leave commented-out code in here.

Does webgl-debug.js still work with WebGL 1.0 contexts after your changes? Or do these changes make it specific to WebGL 2.0? We should keep it working with WebGL 1.0.